core/src/test/scala/io/finch/MissingInstances.scala

```diff
@@ -8,11 +8,18 @@ import com.twitter.util.{Return, Throw, Try}
  * Type class instances for non-Finch types.
  */
 trait MissingInstances {
+
   implicit def eqTry[A](implicit A: Eq[A]): Eq[Try[A]] = Eq.instance {
```

Comment:
I don't think we need this instance anymore.

Reply:
Yeah, my "Fear Driven Development" :)